src/prepack-node.js

```diff
@@ -85,15 +53,16 @@ export function prepackFile(
     filename: string,
     options: Options = defaultOptions,
     callback: (any, ?{code: string, map?: SourceMap})=>void,
-    errorHandler?: (err: ?Error)=>void) {
+    fileErrorHandler?: (err: ?Error)=>void,
+    errorHandler?: ErrorHandler) {
```

Comment:
I am following an existing pattern here and I even remember previous comments requiring it. But, yes, I agree. Putting this into the options object makes more sense and I'll do it.